tests/connect/standalone-x509-error-0001.phpt

```diff
@@ -29,12 +28,13 @@ $opts = array(
         "allow_self_signed" => false,
         "cafile" => $SSL_DIR . "/ca.pem", /* Defaults to openssl.cafile */
         "capath" => $SSL_DIR, /* Defaults to openssl.capath */
-        "local_cert" => $SSL_DIR . "/src/libmongoc/tests/certificates/client.pem",
+        "local_cert" => $SSL_DIR . "/client.pem",
 );
+
+/* Wrong username */
 $parsed = parse_url(STANDALONE_X509);
 $dsn = sprintf("mongodb://username@%s:%d/%s?ssl=true&authMechanism=MONGODB-X509", $parsed["host"], $parsed["port"], DATABASE_NAME);
 
-
 $m1 = connect($dsn, $opts);
 $m2 = connect($dsn, $opts);
```

Comment:
Thats likely. I added a testcase to the report so I wouldn't loose the magic combination of things, and then kept on going with it trying to decipher the issue and/or find others.